db/db_impl.cc

```diff
@@ -674,17 +683,34 @@ void DBImpl::PersistStats() {
     return;
   }
   size_t stats_history_size_limit = 0;
+  bool persist_stats_to_disk = false;
   {
     InstrumentedMutexLock l(&mutex_);
     stats_history_size_limit = mutable_db_options_.stats_history_buffer_size;
+    persist_stats_to_disk = mutable_db_options_.persist_stats_to_disk;
   }
 
-  // TODO(Zhongyi): also persist immutable_db_options_.statistics
-  {
-    std::map<std::string, uint64_t> stats_map;
-    if (!statistics->getTickerMap(&stats_map)) {
-      return;
-    }
+  std::map<std::string, uint64_t> stats_map;
+  if (!statistics->getTickerMap(&stats_map)) {
+    return;
+  }
+  if (persist_stats_to_disk) {
+    std::string now_micros_string = rocksdb::ToString(now_micros);
+    // make time stamp string equal in length to allow sorting by time
+    now_micros_string.insert(
+        0, kTimestampStringMinimumLength - now_micros_string.length(), '0');
+    int keycount = 0;
+    WriteOptions wo;
```

Comment:
Do we consider to put w.low_pri=true and w.no_slowdown to be true? It's also a good idea to explicitly put wo.sync to be false. We may even consider whether we want to write to WAL.